synapse/federation/sender/__init__.py

```diff
@@ -487,5 +488,18 @@ def send_device_messages(self, destination):
 
         self._get_per_destination_queue(destination).attempt_new_transaction()
 
+    def wake_destination(self, destination: str):
+        """Called when we want to retry sending transactions to a remote.
+
+        This is mainly useful if the remote server has been down and we think it
+        migtht have come back.
+        """
+
+        if destination == self.server_name:
+            logger.info("Not waking up ourselves")
+            return
+
```

Comment:
it feels like either we should consider this happening as a sign of badness elsewhere (and log it at more than info), or decide that it's not worth worrying about at all. The halfway house of logging at info seems odd.